### ISSUE TITLE
avl.c:647:2: warning: ‘where’ may be used uninitialized in this function [-Wmaybe-uninitialized]

### ISSUE THREAD

**Issue by kernelOfTruth** [zfsonlinux/zfs#3609]
```
avl.c:647:2: warning: ‘where’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  avl_insert(tree, new_node, where);
  ^
```

**Comment by kernelOfTruth** [comment]
https://github.com/kernelOfTruth/zfs/tree/zfs_master_18.07.2015_issue-3609

https://github.com/kernelOfTruth/zfs/commit/647aa379070e5848cad175ebb812af7418df4694

**Comment by behlendorf** [comment]
Resolved by 273ff9b5ccc438e3099809c43229cc75503cff68.
